### DISCUSSION TITLE
VRAM requirements for Large model?

### DISCUSSION THREAD

**Discussion by Gushousekai195** [GrandaddyShmax/audiocraft_plus#2] [Q&A]
I know the medium model requires 16gb of vram, but what does the large model require? 24gb?

**Comment by GrandaddyShmax** [comment]
I cannot test it myself, but from what i heard, both medium and large models need less than 16gb of vram

**Comment by r4dm** [comment]
Yes, for example, it takes about 23 GB need for a large model with MultiBand_Diffusion Decoder and 80 sec Duration
